c3po/api/service/feed_service.py

```diff
@@ -0,0 +1,35 @@
+from c3po.db.dao.user import UserPosts
+from c3po.db.dao.artist import ArtistSong
+from c3po.db.common.base import session_scope
+
+from logging import getLogger
+from datetime import datetime, timedelta
+
+LOG = getLogger(__name__)
+DEFAULT_TIME_PERIOD = timedelta(days=7)
+
+class FeedService:
+
+    @staticmethod
+    def get_from_to(from_=datetime.now(), to_=datetime.now() - DEFAULT_TIME_PERIOD):
```

Comment:
Can we rename to something else? Like `get_posts_in_interval`? The current name suggests we're fetching a time interval rather than posts in the interval.

Reply:
Cool.